base/src/Module.cpp

```diff
@@ -119,12 +119,13 @@ Module::Module(Kind nature, string name, ModuleProps _props) : mRunning(false),
 
 	pacer = boost::shared_ptr<PaceMaker>(new PaceMaker(_props.fps));
 	auto tempId = getId();
-	mProfiler.reset(new Profiler(tempId, _props.logHealth, _props.logHealthFrequency, [&]() -> std::string {
-		if(!mpFrameFactory.get()){
-			return "";
-		}
-		 return mpFrameFactory->getPoolHealthRecord(); 
-	}));
+	mProfiler.reset(new Profiler(tempId, _props.logHealth, _props.logHealthFrequency, [&]() -> std::string
```

Comment:
Is this a real change or just formating change ? if it is formatting only then please revert it.

Reply:
formatting change